### DISCUSSION TITLE
Gizmo status LEDs will fade out and never come back on

### DISCUSSION THREAD

**Discussion by katekramerhsv** [gizmo-platform/gizmo#39] [Q&A] [answered]
I've been attempting to assist a school at my hub with their gizmo, but I'm having no luck following the documentation or the other Q&As. 

When we started the call, the battery indicator was red, and the network and field lights had no lights. I've had the team install the latest firmware onto the gizmo and attempt a rebind. Now the network light will turn on for a second or two and then fade out and never come back on again. 

I've gone through the Team_experience_v6 doc as well as Gizmo Troubleshooting Guide – Rev 1.0. 
Could we be battling a hardware issue on the gizmo itself, I doubt it since the other indicator LEDs for the other 7 lights are working, but want to be able to rule it out.

**Comment by barulicm** [comment, marked as answer]
This sounds like your battery is dead. A red battery light indicates a battery voltage below 6.5V and the other light pattern sounds like your gizmo is attempting to start up from a dead battery and the voltage drops below the minimum operating voltage once it tries to power on significant systems, like the radio.

Can you check that you see this behavior on a fully charged battery?

**Reply by katekramerhsv** [reply]
I think this got us past the initial issue. We did end up replacing the system processor on the board this weekend at our hub practice day and were able to get the team moving along. Thanks for the input!

**Comment by Josiahaustin** [comment]
update; we have charged battery on top of everything stated above that we did and now when we try to install files it doesn't work and got to the point of where we cannot find it on the computer it still completely will not move motors ever

**Reply by Vaelatern** [reply]
Have you tried programming with the battery disconnected?

Fun fact: if the board is powered, the Pico will not actually power off, and can't be put into programming mode.

**Comment by Josiahaustin** [comment]
We got it working and I'm not really sure if it was or wasn't the board because it only worked after we replaced it
